src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ActionPlanSessionController.kt

```diff
@@ -48,21 +50,42 @@ class ActionPlanSessionController(
   }
 
   @GetMapping("/action-plan/{id}/appointments")
-  fun getSessions(
+  fun getSessionsForActionPlan(
     @PathVariable(name = "id") actionPlanId: UUID
   ): List<ActionPlanSessionDTO> {
 
-    val actionPlanSessions = actionPlanSessionsService.getSessions(actionPlanId)
+    val actionPlan = actionPlanService.getActionPlan(actionPlanId)
+    val actionPlanSessions = actionPlanSessionsService.getSessions(actionPlan.referral.id)
+    return ActionPlanSessionDTO.from(actionPlanSessions)
+  }
+
+  @GetMapping("/referral/{id}/sessions")
```

Comment:
just noticed this has a slight change compared to the previous url

/appointments vs /sessions

is that intentional?

Reply:
This was something you added I think. I'm not sure it's actually being used at the moment? I can update it to appointments if you want?